.github/workflows/acceptance.yml

```diff
@@ -188,40 +147,19 @@ jobs:
           run: |
             yarn install --frozen-lockfile
 
-        - name: Install Python Dependencies
-          env:
-            PGPASSWORD: postgres
-          run: |
-            python setup.py install_egg_info
-            pip install wheel # GitHub Actions does not have this installed by default (unlike Travis)
-            pip install -U -e ".[dev]"
-
-        - name: Start devservices
-          run: |
-            sentry init
-            sentry devservices up postgres redis clickhouse snuba
-
         - name: webpack
           run: |
             yarn webpack --display errors-only
 
-        # Setup custom pytest matcher, see https://github.com/actions/setup-node/issues/97
-        - name: Add pytest log matcher
+        - name: Run acceptance tests (#${{ steps.setup.outputs.matrix-instance-number }} of ${{ strategy.job-total }})
           if: always()
           run: |
-            echo "::remove-matcher owner=pytest::"
-            echo "::add-matcher::.github/pytest.json"
-
-        - name: Run acceptance tests (#${{ steps.config.outputs.matrix-instance-number }} of ${{ strategy.job-total }})
-          if: always()
-          run: |
-            mkdir -p ${{ steps.config.outputs.acceptance-dir }}
-            mkdir -p ${{ steps.config.outputs.acceptance-dir }}-mobile
-            mkdir -p ${{ steps.config.outputs.acceptance-dir }}-tooltips
-            [ "$GITHUB_REF" = "refs/heads/master" ] && export PYTEST_SENTRY_ALWAYS_REPORT=1
+            mkdir -p ${{ steps.setup.outputs.acceptance-dir }}
```

Comment:
Could we just commit these directories so we don't have to worry about this kind of stuff? Like, toss an empty gitignore file into them and push.

Reply:
I'd prefer to leave this for now, I will be doing more work with these in the future for dark mode.